doc/user-manual/language/foreign-function-interface.lagda.rst

```diff
@@ -235,9 +235,9 @@ Level-polymorphic types
 
 :ref:`Level-polymorphic types <universe-levels>` face a similar problem to
 polymorphic functions. Since Haskell does not have universe levels the Agda
-type will have more arguments than the corresponding type. This can be solved
+type will have more arguments than the corresponding Haskell type. This can be solved
 by defining a Haskell type synonym with the appropriate number of phantom
-arguments. For instance
+arguments. For instance:
 
```

Comment:
I think adding a comma `,` is more consistent with the previous part (Polymorphic functions).

Reply:
I'm all for consistency, but neither commas nor colons are used consistently in the document (nor does my pull request fix it). That said, there are more colons than commas between a sentence part and a code snippet.

Reply:
Alright, then consistency will be another issue to solve next time.